ctapipe/instrument/camera/tests/test_description.py

```diff
@@ -1,7 +1,7 @@
 from ctapipe.instrument import CameraDescription
 
 
-def test_known_camera_names():
+def test_known_camera_names(camera_geometries):
```

Comment:
This argument doesn't seem to be used

Reply:
The test asserts that at least 5 geometries are known, the fixture ensures that at least 5 geometries have been downloaded, so they are known.

Reply:
But `cams` is from `CameraDescription.get_known_camera_names()`, `camera_geometries` doesn't look like its used...

Reply:
Oh, so `from_name` will download the files if they aren't already, so that `CameraDescription.get_known_camera_names()` can find them?

Reply:
It seems a bit of a convoluted way to ensure this for the test (not explicit)

Reply:
Yes, agrees

Reply:
We could just remove the `get_known_camera_names` function.

It is not used in any actual analysis and most examples use specific cameras and not something like `get_known_camera_names()[0]`.

Reply:
I like the `get_known_camera_names()` it's a nice feature to be able to see what cameras you can construct. I would not like to remove it.   My original suggestion was to have an index database table (just a FITS table with filename, type, etc in columns) on the server, and you can use that to query what files are there without fetching the data

Reply:
Yes, this was my idea as well. Having this indexes only for things like cameras might me a good compromise to allow full indexing of the server, which was not wished by @bregeon

Reply:
Shall we merge as as and leave the index files to the planned full restructuring of the test data sets?

Reply:
yes, that's fine with me